packages/toolpad-core/package.json

```
@@ -29,14 +29,14 @@
   ],
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/mui/mui-toolpad.git",
+    "url": "https://github.com/mui/toolpad.git",
```

Wrong here

```suggestion
    "url": "git+https://github.com/mui/toolpad.git",
```

https://publint.dev/@toolpad/core@0.6.0

---

> This looks wrong, no?

But it's not 😄  https://docs.npmjs.com/cli/v10/configuring-npm/package-json#repository

> Specify the place where your code lives. This is helpful for people who want to contribute. If the git repo is on GitHub, then the npm repo command will be able to find you.
> 
> Do it like this:
> 
> ```json
> {
>   "repository": {
>     "type": "git",
>     "url": "git+https://github.com/npm/cli.git"
>   }
> }
> ```
> 
> The URL should be a publicly available (perhaps read-only) URL that can be handed directly to a VCS program without any modification. It should not be a URL to an html project page that you put in your browser. It's for computers.

---

Opening https://github.com/mui/toolpad/pull/4113 to correct